src/client/Microsoft.Identity.Client/ApiConfig/AcquireTokenForClientParameterBuilder.cs

```diff
@@ -69,6 +70,29 @@ public AcquireTokenForClientParameterBuilder WithSendX5C(bool withSendX5C)
             return this;
         }
 
+        /// <summary>
+        /// Specifies to use the Managed Identity to fetch the token from the Managed Identity Endpoint.
+        /// For more details see https://aka.ms/msal-net-managed-identity.
+        /// </summary>
+        /// <returns>The builder to chain the .With methods</returns>
+        public AcquireTokenForClientParameterBuilder WithManagedIdentity(string userAssignedClientIdOrResourceId = null)
```

Comment:
Update the comment to mention the parameter for user identities and that system identity will be used without it.